<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowWidget</class>
 <widget class="QWidget" name="BorrowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="UserInfoView" name="m_userInfoView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_logoutButton">
       <property name="styleSheet">
        <string notr="true">background-color: red; color: white;</string>
       </property>
       <property name="text">
        <string>退出登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="BookInfoViewForReader" name="m_bookInfoView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserInfoView</class>
   <extends>QTableView</extends>
   <header>view/userinfoview.h</header>
  </customwidget>
  <customwidget>
   <class>BookInfoViewForReader</class>
   <extends>QTableView</extends>
   <header>view/bookinfoviewforreader.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
